arbnode/transaction_streamer.go

```diff
@@ -601,7 +603,7 @@ func (s *TransactionStreamer) SequenceDelayedMessages(ctx context.Context, messa
 
 func (s *TransactionStreamer) GetGenesisBlockNumber() (uint64, error) {
 	// TODO: when block 0 is no longer necessarily the genesis, track this and update core.NitroGenesisBlock
```

Comment:
I've opened the following Geth PR which includes your changes + some stuff I did to completely remove `core.NitroGenesisBlock` so that we can remove this `TODO`
- https://github.com/OffchainLabs/go-ethereum/pull/103